### DISCUSSION TITLE
Problems running compiled V 5.0.1 on Windows

### DISCUSSION THREAD

**Discussion by chubun** [assimp/assimp#3977] [Q&A]
I downloaded and ran the Windows installer assimp 3.3.1 from assimp.org and everything worked nicely.  However, that version does not export gltf2 (there are some related discussions here about -fgltf2 not working).  So I downloaded the source and compiled using VS 2019.  I followed the instructions on the ReadMe.md page:
1. Open a command prompt
2. mkdir build
3. cd build
4. cmake ..
5. cmake --build .

However, the compiled V 5.0.1 reports many errors that the older version does not.  For examples,

1) Duplicate ID error:
=== Old version:
c:\temp>"d:\Program Files\Assimp\bin\x64\assimp.exe" export Tile_10.dae test.gltf
assimp export: select file format: 'gltf' (GL Transmission Format)
Launching asset import ...           OK
Validating postprocessing flags ...  OK
Importing file ...                   OK
   import took approx. 0.01100 seconds

Launching asset export ...           OK
Exporting file ...                   OK
   export took approx. 0.00200 seconds

**assimp export: wrote output file: test.gltf**

=== V 5.0.1
c:\temp>c:\bin\assimp.exe export W:\temp\Tile_10.dae test.gltf
param = test.gltf
assimp export: select file format: 'gltf2' (GL Transmission Format v. 2)
Launching asset import ...           OK
Validating postprocessing flags ...  OK
Importing file ...                   OK
   import took approx. 0.01300 seconds

Launching asset export ...           OK
Failed to write file
**ERROR: GLTF: two objects with the same ID exist**

2) Unsuported file format:
=== Old version
c:\temp>"d:\Program Files\Assimp\bin\x64\assimp.exe" export W:\temp\Tile_10_L21.mtl test.gltf
assimp export: select file format: 'gltf' (GL Transmission Format)
Launching asset import ...           OK
Validating postprocessing flags ...  OK
Importing file ...                   OK
   import took approx. 0.01800 seconds

Launching asset export ...           OK
Exporting file ...                   OK
   export took approx. 0.00100 seconds

**assimp export: wrote output file: test.gltf**

=== Version 5.0.1
c:\temp>c:\bin\assimp.exe export W:\temp\Tile_10_L21.mtl test.gltf
param = test.gltf
assimp export: select file format: 'gltf2' (GL Transmission Format v. 2)
Launching asset import ...           OK
Validating postprocessing flags ...  OK
**ERROR: Failed to load file: No suitable reader found for the file format of file "W:\temp\Tile_10_L21.mtl".**

Maybe I miss some flags for the compiler?

**Comment by chubun** [comment]
Maybe this should go under "Issues"?  How do I delete the original post?

**Comment by kimkulling** [comment]
There is no option to close it.
